lib/code_corps/github.ex

```diff
@@ -2,10 +2,37 @@ defmodule CodeCorps.Github do
 
   alias CodeCorps.{User, Repo}
 
+  @token_url ""
+  @client_id ""
+  @client_secret ""
+
```

Comment:
`client_id` and `_secret` will be environment variables which we will get when we register CodeCorps as a github application. It may even make sense not to have them as module variables, so we can inject them in tests. 

Either way, assume the environment variables `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` exist, are loaded during config and stored into a config key.

For `@token_url`, this should simply be `https://github.com/login/oauth/access_token`